config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_span_attributes.conf

```diff
@@ -103,7 +115,11 @@ commands = [
               display_name: Protocol,
               scope: EVENT,
               sources: [QS],
+              internal: true,
```

Comment:
We should not add `internal: true` to existing attributes, because they were already exposed before